catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemValidator.java

```diff
@@ -26,35 +24,52 @@
  * required by the {@link ddf.catalog.content.StorageProvider}.
  */
 public class ContentItemValidator {
-
     private static final Pattern QUALIFIER_PATTERN = Pattern.compile("\\w[a-zA-Z0-9_\\-]+");
```

Comment:
@brendan-hofmann Stripping dashes from the UUID predates the use of Solr.  I don't know why they were stripped originally.  Maybe @shaundmorris remembers.